src/Command/CheckCommand.php

```diff
@@ -76,19 +76,24 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $drupalFinder = new DrupalFinder();
-        $path = realpath($input->getArgument('path'));
 
-        if (!$path || !file_exists($path)) {
-            $output->writeln(sprintf('<error>%s does not exist</error>', $input->getArgument('path')));
-            return 1;
+        $paths = [];
+        foreach ($input->getArgument('path') as $path) {
+            $realPath = realpath($path);
+            if (!$realPath) {
```

Comment:
The `file_exists()` check was removed since `realpath()` already performs that check